website/docs/language/meta-arguments/module-providers.html.md

```diff
@@ -36,9 +36,10 @@ module "example" {
 
 ## Default Behavior: Inherit Default Providers
 
-The `providers` argument is optional. If you omit it, a child module inherits
-all of the _default_ provider configurations from its parent module. (Default
-provider configurations are ones that don't use the `alias` argument.)
+If the child module does not declare any configuration aliases, the `providers`
```

Comment:
Since "configuration aliases" is a new concept here, would be good to link or define.

Reply:
Good idea! The concept isn't new (nor is the requirement, it just wasn't enforced), but I do like linking between the pages to make things more discoverable in general.